src/services/user-service.js

```diff
@@ -1,39 +1,51 @@
+const LRUCache = require('lru-native');
 const { log } = require('steno');
 
 const User = require('../models/UserModel');
-const cache = require('./cache-service.js');
 
-module.exports = {
-    get(uid) {
-        const cachedUser = cache.get(uid);
+const cache = new LRUCache({
+    maxElements: 1000,
+    maxAge: 120 * 60 * 1000, // 2 hours
+    size: 100
```

Comment:
I was thinking about it. But:
1. I doubt we want to configure it "on fly" or during the "compilation"
2. In general it should be moved to a separate service (cache service? data-storage service?)

If it really smells - no problem, can move to config